README.rst

```diff
@@ -208,7 +208,7 @@ Example: CPU usage should be less than 90% but current value is at 95%.
 
 One of the metrics defined in ``OPENWISP_MONITORING_CRITICAL_DEVICE_METRICS``
 has a value which is not in the expected range
-(threshold value crossed).
+(AlertSettings value crossed).
 
```

Comment:
applies to here as well, see how I have edited it in the branch where this will be merged